src/main/java/vazkii/botania/common/Botania.java

```diff
@@ -285,6 +287,9 @@ private void serverAboutToStart(FMLServerAboutToStartEvent event) {
 					+ " \"Do not Override\". Whoever had the brilliant idea of overriding it needs to go"
 					+ " back to elementary school and learn to read. (Actual classname: " + clname + ")");
 		}
+		if (Botania.gardenOfGlassLoaded) {
+			WorldTypeUtil.setupForDedicatedServer((DedicatedServer) event.getServer());
```

Comment:
this seems unsafe, what if it's an integrated server?

Reply:
true, fixed it :)